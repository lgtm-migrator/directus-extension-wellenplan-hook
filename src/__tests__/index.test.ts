import index from "../index";

// TODO: add meaningful tests
describe("wellenplan", () => {
  describe("hook", () => {
    describe("index", () => {
      it("should define a hook", () => {
        expect(index).toMatchSnapshot();
      });
    });
  });
});
