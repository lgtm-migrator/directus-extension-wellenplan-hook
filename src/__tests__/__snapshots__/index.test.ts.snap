// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wellenplan hook index should define a hook 1`] = `[Function]`;
